activerecord/lib/active_record/connection_adapters/postgresql/oid/hstore.rb

```diff
@@ -24,6 +24,8 @@ def deserialize(value)
           def serialize(value)
             if value.is_a?(::Hash)
               value.map { |k, v| "#{escape_hstore(k)}=>#{escape_hstore(v)}" }.join(", ")
+            elsif value.respond_to?(:to_unsafe_h)
```

Comment:
Why do we allow unpermitted params to be persisted?
IMO, we should only persist params that have been permitted with strong parameters API.